docs/pages/server-access/guides/vscode.mdx

```diff
@@ -120,21 +139,20 @@ Host node000.foo.example.com
 When finished, save the file. If the added host doesn't automatically appear in
 the list, select the Refresh button.
 
-## Step 3/3. Start a Remote Development Session
+## Step 3/3. Start a Remote Development session
 
-Start a remote development session by right
-clicking on any host added above and selecting either "Connect to..." option:
+Start a Remote Development session by right-clicking on any host added above and selecting either "Connect to..." option:
```

Comment:
I tested this section too and it worked.  I did run into the vscode-remote error `Failed to parse remote port from server output`, but this was solved by setting the following options in vscode:
```
"remote.SSH.useLocalServer": false,
"remote.SSH.showLoginTerminal": true,
```

I'm not sure getting into the nitty gritty of vscode configuration like this is within scope of this article, but if anyone else testing this gets the same error, we might want to put it in.

Reply:
Thanks! I suppose we can wait to see if others are having the same issue.